### PR TITLE
[Enhancement] Remove sensitive info of broker load from audit log

### LINKED ISSUE #13324: Sensitive info should be moved from log
## Enhancement
The log of StarRocks may include some sensitive information, like username, password.

### PR DESCRIPTION
**Pull request by rickif**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes https://github.com/StarRocks/starrocks/issues/13324

## Problem Summary(Required) ：

This PR removes access key/secret from audit log of broker load. The new output would be like
```
2022-11-15 11:53:26,539 [query] |Client=127.0.0.1:56418|User=root|AuthorizedUser='root'@'%'|ResourceGroup=default_wg|Catalog=default_catalog|Db=db0|State=ERR|ErrorCode=|Time=1|ScanBytes=0|ScanRows=0|ReturnRows=0|CpuCostNs=0|MemCostBytes=0|StmtId=4|QueryId=0f279750-6499-11ed-90d8-52243889412b|IsQuery=false|feIp=172.17.0.1|Stmt=LOAD LABEL `db0`.`11label_test3`(DATA INFILE ('gs://test_simo/4097.csv') INTO TABLE tbl_test COLUMNS TERMINATED BY ',' (key1, key2))WITH BROKER hdfs_broker ("fs.s3a.access.key"  =  "***", "fs.s3a.secret.key"  =  "*XXX", "fs.s3a.endpoint"  =  "storage.googleapis.com")|Digest=|PlanCpuCost=0.0|PlanMemCost=0.0
```

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
- [ ] I have added documentation for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 8 / 31 (25.81%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|0|3|00.00%|[87, 88, 89]|
|:large_blue_circle:|com/starrocks/sql/ast/LoadStmt.java|0|19|00.00%|[284, 285, 286, 287, 290, 292, 293, 294, 296, 297, 298, 299, 301, 302, 305, 306, 307, 308, 310]|
|:large_blue_circle:|com/starrocks/qe/ConnectProcessor.java|2|3|66.67%|[182]|
|:large_blue_circle:|com/starrocks/common/util/PrintableMap.java|6|6|100.00%|[]|

**Comment by rickif** [comment]
@mergifyio rebase

**Comment by rickif** [comment]
run all

**Comment by rickif** [comment]
https://github.com/Mergifyio rebase

**Comment by rickif** [comment]
@Mergifyio rebase

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 27 / 33 (81.82%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/analysis/BrokerDesc.java|0|1|00.00%|[104]|
|:large_blue_circle:|com/starrocks/qe/ConnectProcessor.java|2|3|66.67%|[182]|
|:large_blue_circle:|com/starrocks/sql/analyzer/AST2SQL.java|15|19|78.95%|[121, 122, 123, 126]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/sql/ast/LoadStmt.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/common/util/PrintableMap.java|6|6|100.00%|[]|

File: fe/fe-core/src/test/java/com/starrocks/analysis/LoadStmtTest.java [hunk 1]
```diff
@@ -88,4 +88,10 @@ public void testBrokerLoad() {
         analyzeSuccess("LOAD LABEL test.testLabel (DATA INFILE(\"hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file\") INTO TABLE `t0`) WITH BROKER (\"username\"=\"sr\") PROPERTIES (\"strict_mode\"=\"true\")");
         analyzeSuccess("LOAD LABEL test.testLabel (DATA INFILE(\"hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file\") INTO TABLE `t0`) WITH BROKER (\"username\"=\"sr\")");
     }
+
+    @Test
+    public void testToString() {
+        LoadStmt stmt = (LoadStmt) analyzeSuccess("LOAD        LABEL test.testLabel (DATA INFILE(\"hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file\") INTO TABLE `t0`) WITH BROKER hdfs_broker (\"username\"=\"sr\", \"password\"=\"PASSWORDDDD\") PROPERTIES (\"strict_mode\"=\"true\")");
+        Assert.assertEquals("LOAD LABEL `test`.`testLabel`(DATA INFILE ('hdfs://hdfs_host:hdfs_port/user/starRocks/data/input/file') INTO TABLE t0)WITH BROKER hdfs_broker (\"password\"  =  \"*XXX\", \"username\"  =  \"sr\")PROPERTIES (\"strict_mode\" = \"true\")", stmt.toString());
```
Comment (chaoyli) [inline fe/fe-core/src/test/java/com/starrocks/analysis/LoadStmtTest.java:95]: What's the `*XXX`?  You can use `xxx` instead.

Comment (rickif) [reply fe/fe-core/src/test/java/com/starrocks/analysis/LoadStmtTest.java:95]: https://github.com/StarRocks/starrocks/blob/a1c41554cceced937e3cd4fb3a57d3a9485c567c/fe/fe-core/src/main/java/com/starrocks/common/util/PrintableMap.java#L87-L89
It's added by the `PrintableMap.toString`. How about replacing it with `***` ?

File: fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java [hunk 2]
```diff
@@ -177,7 +177,10 @@ public void auditAfterExec(String origStmt, StatementBase parsedStmt, PQueryStat
 
         ctx.getAuditEventBuilder().setFeIp(FrontendOptions.getLocalHostAddress());
 
-        if (ctx.getState().isQuery() && containsComment(origStmt)) {
+        if (!ctx.getState().isQuery() && (parsedStmt != null && parsedStmt.needAuditEncryption())) {
+            // Some information like username, password in the stmt should not be printed.
+            ctx.getAuditEventBuilder().setStmt(parsedStmt.toString());
```
Comment (HangyuanLiu) [inline fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java:182]: toString is an incomplete interface. Unified to sql functionality in AST2SQL

Comment (rickif) [reply fe/fe-core/src/main/java/com/starrocks/qe/ConnectProcessor.java]: OK. It has been replaced by `AST2SQL.toString(parsedStmt)`.

**Review by HangyuanLiu — DISMISSED** [review]

**Comment by rickif** [comment]
run starrocks_fe_unittest

**Comment by rickif** [comment]
@mergifyio rebase

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 27 / 32 (84.38%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/qe/ConnectProcessor.java|2|3|66.67%|[191]|
|:large_blue_circle:|com/starrocks/sql/analyzer/AST2SQL.java|15|19|78.95%|[121, 122, 123, 126]|
|:large_blue_circle:|com/starrocks/analysis/LabelName.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/common/util/PrintableMap.java|7|7|100.00%|[]|

**Review by chaoyli — APPROVED** [review]

**Review by meegoo — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**